templates/templates/base.html

```diff
@@ -25,17 +25,17 @@
     <meta name="theme-color" content="#E95420">
     <meta name="twitter:account_id" content="4503599627481511">
     <meta name="twitter:site" content="@ubuntu">
-    <meta property="og:type" content="website">
-    <meta property="og:url" content="https://ubuntu.com{{ request.get_full_path }}">
-    <meta property="og:site_name" content="Ubuntu">
+    <meta prefix="og: http://ogp.me/ns#" property="og:type" content="website">
```

Comment:
I believe you can put this namespace on the `<html>` element, can't you? https://ogp.me/